packages/lib-user/src/components/UserStats/components/ProfileHeader.js

```diff
@@ -0,0 +1,139 @@
+import { Box, Image } from 'grommet'
+import { number, string } from 'prop-types'
+import styled from 'styled-components'
+import { SpacedText, withResponsiveContext, ZooniverseLogo } from '@zooniverse/react-components'
```

Comment:
Should this be refactored as individual deep imports?

Reply:
I'm not sure of a great way to test the difference yet. I think this is a good read about barrel files though: https://vercel.com/blog/how-we-optimized-package-imports-in-next-js. If time before Shaun leaves, let's chat in standup about if he had success with deep imports in Community Catalog. I'll also do some more research into ways to tests various app size in FEM!